src/wallet/rpcwallet.cpp

```diff
@@ -869,18 +871,42 @@ UniValue sc_create(const UniValue& params, bool fHelp)
     if (params.size() > 6)
     {
         const std::string& inputString = params[6].get_str();
-        std::vector<unsigned char> wMbtrVkVec;
-        if (!Sidechain::AddScData(inputString, wMbtrVkVec, SC_VK_SIZE, true, error))
+        // it is optional, non-empty if set
+        if (!inputString.empty())
```

Comment:
so if empty, do we need to return error?
Seems currently we just ignore the field.

Reply:
We check only contents if any